src/App.scss

```
@@ -1 +1,40 @@
-// styles go here
+.App {
+
+  &__header {
+    position: sticky;
+    top: 10px;
+  }
+
+  &__people {
+    text-align: center;
+  }
+
+  &__home {
+    text-align: center;
+  }
+
+  &__unknown {
+    text-align: center;
+    color: red;
+    font-size: 25px;
+    font-weight: bold;
+  }
+}
+
+.navbar {
+
+  &__list {
+    display: flex;
+    justify-content: right;
+    padding-right: 35px;
+  }
+
+  &__item {
+    display: block;
+    margin: 25px;
+    color: red;
+    text-decoration: none;
+    font-size: 25px;
+    font-weight: bold;
+  }
+}
```

it would nice to group selector's and properties
```suggestion
  &__item {
    display: block;
    margin: 25px;
    
    color: red;
    text-decoration: none;
    font-size: 25px;
    font-weight: bold;
  }
}
```